apps/meteor/app/lib/server/functions/saveUser.ts

```diff
@@ -356,7 +354,7 @@ export const saveUser = async function (userId, userData) {
 		delete userData.setRandomPassword;
 	}
 
-	if (!userData._id) {
+	if (!('_id' in userData)) {
```

Comment:
This is not exactly the same behavior as we had previously. We need to check also if it's present but it's undefined.

Reply:
This one was quite tricky. Not relying only on the `_id` property being in `userData` caused some typecheck issues. So I had to use `as ICreateUserParams` to avoid this. Couldn't think of a better approach to keep this check.